integration-tests/api/helpers/claim-seeder.js

```diff
@@ -0,0 +1,107 @@
+const {
+  ClaimOrder,
+  Order,
+  LineItem,
+  Fulfillment,
+  Return,
+} = require("@medusajs/medusa");
+
+module.exports = async (connection, data = {}) => {
```

Comment:
Adding a claim seeder, which adds a claim that contains both return and fulfillments